### PR TITLE
Adapt About-us-page to also include information about the exam feature

### PR DESCRIPTION
**Pull request by JohannesStoehr**
### Checklist
- [x] All described elements are implemented
- [x] Client: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html).
- [x] Client: I added multiple screenshots/screencasts of my UI changes
- [x] Client: I translated all the newly inserted strings into German and English

### Motivation and Context
Artemis has grown and inludes an exam mode as well. This should also be shown in the about-us-page.
Additional links to all features and a guide are linked there as well.

### Description
A second section is now added to the page after the details about all supported exercise types.
Additionally the title of the page as shown by the browser should not be "[object Object]".

### Steps for Testing

1. Log in to Artemis
2. Navigate to the About-us-page linked on the bottom left corner of every ArTEMiS page.
3. Check the new section about the exam mode:
- That all necessary information is present
- That the German AND Englisch texts are concise and correct (you can change the language by clicking on your name on the top right)
4. Check that the title of the page is meaningful in both languages

### Screenshots
Bildschirmfoto 2021-03-19 um 20 55 13

### CODE REVIEW

File: src/main/webapp/i18n/de/aboutUs.json [hunk 1]
```diff
@@ -12,6 +12,23 @@
...
+                "randomSelectionOfVariants": "Mehrere Varianten von Aufgaben können erstellt und zufällig den Studenten zugewiesen werden",
+                "fullFeatureOverview": {
+                    "description": "Gesamter Überblick über alle Features:",
+                    "studentLink": "Überblick für Studenten",
+                    "instructorLink": "Überblick für Instructors"
+                },
+                "instructions": {
+                    "description": "Anleitung für das Klausurfeature:",
+                    "studentLink": "Anleitung für Studenten",
+                    "instructorLink": "Anleitung für Instructors"
```
Comment (Strohgelaender) [inline src/main/webapp/i18n/de/aboutUs.json:28]:
```suggestion
                    "instructorLink": "Überblick für Dozenten"
                },
                "instructions": {
                    "description": "Anleitung für das Klausurfeature:",
                    "studentLink": "Anleitung für Studenten",
                    "instructorLink": "Anleitung für Dozenten"
```
Instructors are called "Dozenten" in other german translation strings, this should be consistent.

**Review by SimonPannek — APPROVED** [review]

**Review by ivohashamov — APPROVED** [review]
Tested on ts2, looks good!

**Comment by krusche** [comment]
@JohannesStoehr 2 client tests fail. Can you have a look? (please ask some other developers in case you need help)

File: src/main/webapp/app/core/about-us/about-us.component.html [hunk 2]
```diff
@@ -12,6 +12,37 @@ <h6>{{ 'aboutUsOverview.exercises.title' | artemisTranslate }}</h6>
             <li>{{ 'aboutUsOverview.exercises.detail.textExercises' | artemisTranslate }}</li>
             <li>{{ 'aboutUsOverview.exercises.detail.uploadExercises' | artemisTranslate }}</li>
         </ul>
+        <h6>{{ 'aboutUsOverview.exam.title' | translate }}</h6>
```
Comment (FrankeLukas) [inline src/main/webapp/app/core/about-us/about-us.component.html:15]: Try using `artemisTranslate` instead of `translate` here.

**Review by ga62suz — DISMISSED** [review]
For the most part looks good.

But there is one picture that is broken (Linus Michel).

**Review by PatZintel — DISMISSED** [review]
Tested successfully

Linus Michel's picture is broken though

**Review by real-dmy — DISMISSED** [review]
Tested successfully. 
One image ( Linus Michel) is not showing up.

**Review by ArtemTsurikov — COMMENTED** [review]
Looks good.

**Comment by Strohgelaender** [comment]
The broken picture is not related to the changes here, so my comment on https://github.com/ls1intum/Artemis/commit/4334830ecbd12a214055284982aef591dde26ded

**Review by cremertim — COMMENTED** [review]
Seems to work fine for me. One Photo is not displayed properly 

Although, the translation to german might be updated another time

Programmierübungen mit Versionskontrolle und automatischer Bewertung _(mit)_  **anhand von** Testfällen und kontinuierlicher Integration

Modellierungsübungen mit halbautomatischer Bewertung unter Verwendung von **Machine Learning** Konzepten _(von Machine Learning)_

Übungen **zum** _(mit)_ Hochladen von Dateien mit **anschließender** manuellen Bewertung

Mehrere Varianten von Aufgaben können erstellt und _(zufällig)_ den Studenten **zufällig** zugewiesen werden

bold are my proposed changes, between the brackets what I removed

File: src/test/javascript/spec/component/course/course.component.spec.ts [hunk 3]
```diff
@@ -180,10 +180,7 @@ describe('CoursesComponent', () => {
     it('Should load next relevant exam', () => {
         const navigateSpy = sinon.spy(router, 'navigate');
         component.nextRelevantCourseForExam = course1;
-        component.nextRelevantExams = exams;
-        const findAllForDashboardStub = stub(courseService, 'findAllForDashboard');
-        findAllForDashboardStub.returns(of(new HttpResponse({ body: courses, headers: new HttpHeaders() })));
-
+        component.nextRelevantExams = [exam1];
```
Comment (iozbeyli) [inline src/test/javascript/spec/component/course/course.component.spec.ts:183]: Why is this change necessary for this pull request?

Comment (JohannesStoehr) [reply src/test/javascript/spec/component/course/course.component.spec.ts:183]: That was a flaky test case as discussed here: https://ls1tum.slack.com/archives/G6TCVL6HL/p1616406625032800?thread_ts=1616405571.030700&cid=G6TCVL6HL
So I fixed it together with Nicolas to get all tests passing again as demanded by Stephan https://github.com/ls1intum/Artemis/pull/3056#issuecomment-803630427

**Review by krusche — APPROVED** [review]
